### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by GrassIsNotBlue** [RedShyGuy/Vapecord-ACNL-Plugin#52] [Ideas]
I have some suggestions/ideas for an update, if this is possible, than i would love to see this in acnl.   
1: some way to open your locker while on the island. There is already something to open your locker if you edit a design, but that isn't possible on the island. Maybe something that if you open your encyclopedia, you go to the locker instead of the encyclopedia menu. Also this hat a big advantahe because the encyc. Menu has an insects, fish, and seafood submenu. Than you could open 3 different menus.
2: more than 4 people on the island.         
3: a button to go to another island, without leaving and then going back in the boat. 
4: touching items in your inventory to see their item IDs. 
5: being able to place furniture outside. 
6: real time map editor for island

**Comment by RedShyGuy** [comment]
1. I can make something like that yes
2. Not possible, as that is completely server sided
3. Way too difficult, I doubt I could make something like that
4. Already a thing, enable Player Info and press on a item, on the top right will be the ID
5. Not possible, the game isn't built to do that
6. If you mean editing the Acres that's also already a thing, though editing them while on the island won't do anything, that's why you need to edit them before going to the island
